src/jit/codegenlegacy.cpp

```diff
@@ -18880,32 +18899,69 @@ regMaskTP CodeGen::genCodeForCall(GenTreeCall* call, bool valUsed)
 
                     if (vtabOffsOfIndirection != CORINFO_VIRTUALCALL_NO_CHUNK)
                     {
+                        if (isRelative)
+                        {
+#if defined(_TARGET_ARM_)
+                            unsigned offset = vtabOffsOfIndirection + vtabOffsAfterIndirection;
+
+                            // ADD vptrReg1, REG_CALL_IND_SCRATCH, vtabOffsOfIndirection + vtabOffsAfterIndirection
+                            getEmitter()->emitIns_R_R_I(INS_add, EA_PTRSIZE, vptrReg1, vptrReg, offset);
+#else
+                            _ASSERTE(false);
+#endif
```

Comment:
Actually, this (and below) should be `unreached()`

---
In reply to: [129660546](https://github.com/dotnet/coreclr/pull/12168#discussion_r129660546) [](ancestors = 129660546)